### DISCUSSION TITLE
A few issues

### DISCUSSION THREAD

**Discussion by madeupname24** [aclap-dev/video-downloadhelper#613] [Help and Q&A]
### Your question

Using Firefox.
The play button does nothing when pressed.
When downloading the progress bar just shows a blue bar moving left to right, it doesn't show any actual progress.

I find it really annoying that it takes over my side bar removing my bookmarks and it doesn't put them back when I close VDH.
There is an option at the botton of the side bar that says "show in popup" but all that does is close the side bar.  

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-GB
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\Speedy\\dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_wide_ui": false,
    "use_sidebar": false,
    "open_count_store": 10
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.

> The play button does nothing when pressed.

It only appears to do nothing.  If you are using VLC, and you should be, pressing the green Play button on an entry in the VDH sidebar or menu adds the video to the VLC playlist.  It just queues up the video for playback.  Playback is something you need to do as a separate step, manually.

> When downloading the progress bar just shows a blue bar moving left to right, it doesn't show any actual progress.

That would be an indication of a problem.  If you supply details of the web page on which this occurred, maybe somebody can investigate what's going on.

> I find it really annoying that it takes over my side bar removing my bookmarks and it doesn't put them back when I close VDH.
> There is an option at the botton of the side bar that says "show in popup" but all that does is close the side bar.

I suppose it is annoying that sending the sidebar back to the menu doesn't reinstate whatever you might have had in the sidebar before VDH.  Oh well.  Just put your bookmarks back in the sidebar manually.

But clicking the button at the bottom of the VDH sidebar does more than just close it.  It also re-enables getting the VDH menu from the VDH button on the browser task bar.  Closing the VDH sidebar gives you back the ability to drop down the VDH menu like has always been possible in all previous versions of VDH.  Personally, I don't like the sidebar.  It takes up space from the web page being displayed.  It also interferes with convenient keyboard-only control of VDH.  You can get to the sidebar exclusively from the keyboard.  But you have to hit Tab a ridiculous number of times to get there.  When you use the classic drop down menu instead, this version of VDH allows you to do everything exclusively from the keyboard.  To me, that is the single most important improvement in the product with this new version 9.

So if you want to investigate that progress thing, post some relevant details & let's take it from there.

**Reply by GrampaWildWilly** [reply]
Correction.  The F6 key navigates you into the sidebar.  Well, I guess that blows away one of my objections.  Still, I don't like that it takes up space from the web page.  I don't use the bookmarks sidebar, either, for the same reason.  I like dropping down the VDH menu when I want to use it.  It's just a matter of taste.  Different strokes for different folks.

**Reply by GrampaWildWilly** [reply]
You're using Firefox 115?  Are you still on Windows 7?  My sympathies . . .
